### ISSUE TITLE
Is bladebit dead ?

### ISSUE THREAD

**Issue by piotr-nowicki** [Chia-Network/bladebit#459]
There are no relases, bug fixes, code merge for past 6 months.

Did you abandoned project ?.

**Comment by haorldbchi** [comment]
Not at all, but we're busy with some major updates with plotting at the moment. Simply don't have enough bandwidth to juggle both things

**Comment by EduardoLLMM** [comment]
Waiting for compression upgrade with only 128GB RAM

**Comment by kinomexanik** [comment]
when is the new release? for new plotformat
